website/docs/patterns/button-alignment/partials/overview/overview.md

```diff
@@ -0,0 +1,98 @@
+
+Unconstrained generally refers to a more "static" experience that isn't triggered by a user action. This is often tied to a page-level function or a navigational element.
+
+**Insert some supporting images here**
+
+## Layout and reading patterns
+
+When users read content, the pattern their eyes follow generally adheres to one or more reading pattern. By identifying and familiarizing ourselves  with these patterns we can better understand how the layout within our application, pattern, or component impacts comprehension and interpretation of the content.
+
+The three most common reading patterns are:
+
+- the Gutenberg diagram
+- Z-shaped pattern
+- F-shaped pattern
+
+### Gutenberg diagram
+
+The Gutenberg diagram refers to a pattern the eyes follow when moving or scanning through **evenly distributed, homogenous content.** This pattern is most relevant in text-heavy scenarios like novels and newspapers, but is relevant in a UX context depending on the complexity of the content.
+
+![Visual example of a Gutenberg diagram](/assets/patterns/button-alignment/gutenberg-diagram.png)
+
+#### Characteristics
+
+- The users eyes progress through the content on an **axis of orientation** which moves further away from the start edge on each line or break in the content
+- Content that falls into the weak fallow area recieves less attention
+- This pattern indicates that important content and elements should be placed upon the **reading gravity path**
+- This pattern is most prevalent in content with little to no typographic hierarchy (i.e. long-form paragraphs)
+- This pattern is considered "in harmony" with natural reading direction
+
+### Z-shaped pattern
+
+A Z-shaped pattern refers to a pattern that follows the shape of the letter Z:
+
+1. The users eyes start in the top left and move horizontally to the right
+2. Upon reaching the end (or terminal) of the line, the eyes move diagonally to the next line
+3. The readers eyes end at the bottom right of the content
+
+![Visual example of a Z-shaped pattern](/assets/patterns/button-alignment/z-shaped-pattern.png)
+
+#### Characteristics
+
+- There are _technically_ no fallow areas (areas that recieve less attention or lack comprehension) as in the Gutenberg pattern, though this is not always the case in practice.
+- This type of pattern is most prevalent in task-based functions or sequences of elements; i.e. a form or series of onboarding steps.
+- This pattern assumes that users read through _all_ of the content sequentially.
+
+### F-shaped pattern
+
+An F-shaped pattern loosely follows the shape of the letter F, first identified through eye-tracking and heatmap studies by the [Neilsen/Norman Group](https://www.nngroup.com/articles/f-shaped-pattern-reading-web-content-discovered/).
+
+![Visual example of an F-shaped pattern](/assets/patterns/button-alignment/f-shaped-pattern.png)
+
+**We probably can't use this image directly on our webiste :(**
```

Comment:
Maybe we should remove it to be safe? I think as long as we link to the page on their site which explains this pattern then that's probably fine. (Or we could make our own image somehow.)